moovingon_ai/manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1a02140e-4927-49c9-8442-dff81a18c703",
+  "app_id": "moovingon-ai",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "moovingon.ai is a NOC orchestration and automation platform",
+    "title": "moovingon.ai",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Guidelines",
+        "image_url": "images/moovingon_ai-guidelines.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Events",
+        "image_url": "images/moovingon_ai-events.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Integrations",
+        "image_url": "images/moovingon.ai-integrations.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Overview Dashboard",
+        "image_url": "images/moovingon_ai-overview-dashbard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Notifications",
+      "Category::Incidents",
+      "Category::Event Management",
+      "Category::Alerting",
+      "Category::Issue Tracking",
+      "Offering::Integration",
+      "Submitted Data Type::Events",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
As mentioned in your previous comment, it sounds like you are pulling data out of Datadog. If so, please add the associated tag value from this [list](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). ex: 

Queried Data Type::<fill in>

Reply:
@emarsha94

Added:
Queried Data Type::Events
Submitted Data Type::Events